Formula/libpulsar.rb

```diff
@@ -27,16 +27,13 @@ class Libpulsar < Formula
   uses_from_macos "curl"
 
   def install
-    cd "pulsar-client-cpp" do
-      system "cmake", ".", *std_cmake_args,
-                      "-DBUILD_TESTS=OFF",
-                      "-DBUILD_PYTHON_WRAPPER=OFF",
-                      "-DBoost_INCLUDE_DIRS=#{Formula["boost"].include}",
-                      "-DProtobuf_INCLUDE_DIR=#{Formula["protobuf"].include}",
-                      "-DProtobuf_LIBRARIES=#{Formula["protobuf"].lib}/libprotobuf.dylib"
-      system "make", "pulsarShared", "pulsarStatic"
-      system "make", "install"
-    end
+    system "cmake", ".", *std_cmake_args,
+                    "-DBUILD_TESTS=OFF",
+                    "-DBoost_INCLUDE_DIRS=#{Formula["boost"].include}",
+                    "-DProtobuf_INCLUDE_DIR=#{Formula["protobuf"].include}",
+                    "-DProtobuf_LIBRARIES=#{Formula["protobuf"].lib}/libprotobuf.dylib"
```

Comment:
It looks like we need to link to `libprotobuf.so` for Linux to fix the failed Ubuntu-22.04 tests. I didn't look into the details of Homebrew, but I guess the following solution could work?

```ruby
library_suffix = OS.linux? ? "so" : "dylib"
system "cmake", ".", *std_cmake_args,
# ...
    "-DProtobuf_LIBRARIES=#{Formula["protobuf"].lib}/libprotobuf.#{library_suffix}"
```

@michaeljmarshall

Reply:
Thanks! I'll test that out.

Reply:
That worked.